README.md

```diff
@@ -64,23 +64,36 @@ If you want to use this automation, read the instructions in [App Creation Scrip
+   - Leave **Supported account types** on the default setting of **Accounts in this organizational directory only**.
+   - For the **Redirect URI (optional)**, select Web and enter the base URL for the sample. By default, this sample uses `https://localhost:44322/`.
+1. Click **Register** to create the application.
+1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+1. Select the **Expose an API** section, and:
+   - Select **Add a scope**
+   - Set the Application ID URI to be '<https://<your_tenant_name>/TaskTrackerWebApp-GroupClaims'> (replacing <your_tenant_name> with the name of your Azure AD tenant)
+   - Enter the following parameters
+     - for **Scope name** use `access_as_user`
+     - Keep **Admins and users** for **Who can consent**
+     - in **Admin consent display name** type `Access MyWebAPI as a user`
+     - in **Admin consent description** type `Accesses the MyWebAPI Web API as a user`
+     - in **User consent display name** type `Access MyWebAPI as a user`
+     - in **User consent description** type `Accesses the MyWebAPI Web API as a user`
+     - Keep **State** as **Enabled**
+     - Select **Add scope**
+1. From the **Certificates & secrets** page, in the **Client secrets** section, choose **New client secret**:
+   - Type a key description (for instance `app secret`),
    - Select a key duration of either **In 1 year**, **In 2 years**, or **Never Expires**.
-   - When you save this page, the key value will be displayed, copy, and save the value in a safe location.
-   - You'll need this key later to configure the project in Visual Studio. This key value will not be displayed again, nor retrievable by any other means,
-     so record it as soon as it is visible from the Azure portal.
-1. Configure Permissions for your application. To that extent, in the Settings menu, choose the 'Required permissions' section and then,
-   click on **Add**, then **Select an API**, and type `Microsoft Graph` in the textbox. Then, click on  **Select Permissions** and select **Directory.Read.All**.
+   - When you press the **Add** button, the key value will be displayed, copy, and save the value in a safe location.
+   - You'll need this key later to configure the project in Visual Studio. This key value will not be displayed again, nor retrievable by any other means, so record it as soon as it is visible from the Azure portal.
+1. Configure Permissions for your application. Select the **API permissions** section
+   - Click the **Add a permission** button and then,
+   - Ensure that the **Microsoft APIs** tab is selected
+   - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**
+   - In the **Delegated permissions** section, ensure that the right permissions are checked: **Directory.Read.All**, **Mail.Send**. Use the search box if necessary.

```

Comment:
@AllisonAm : does the sample require Mail.Send? I have not seen it?

Reply:
Updated. Sorry, came from the template!